prestapyt/prestapyt.py

```diff
@@ -211,12 +230,19 @@ def _execute(self, url, method, data=None, add_headers=None):
         request_headers = self.client.headers.copy()
         request_headers.update(add_headers)
 
-        response = self.client.request(
-            method,
-            url,
-            data=data,
-            headers=request_headers,
-        )
+        if self.verbose:
+            currentlevel = HTTPConnection.debuglevel
```

Comment:
Why not HTTPConnection.set_debuglevel(level) ?

Reply:
Ok I got it you override the package in order to affect requests library. 
It is even in the requests doc. Maybe not the cleanest way but OK

:+1:

Reply:
yeah I got that in the requests doc an didn't searched further